algebra/src/bytes.rs

```diff
@@ -222,6 +222,20 @@ impl FromBytes for u64 {
     }
 }
 
+impl ToBytes for usize {
```

Comment:
I won't implement serialization/deserialization methods for usize, by converting it as default to an u64: it highly depends on the use case, an maybe an u64 can be oversized, so I would remove them

Reply:
Implementation for usize is removed